consul_exporter.go

```diff
@@ -96,10 +104,22 @@ func NewExporter(uri, kvPrefix, kvFilter string, healthSummary bool, consulTimeo
 		return nil, fmt.Errorf("invalid consul URL: %s", uri)
 	}
 
+	tlsConfig, err := consul_api.SetupTLSConfig(&consul_api.TLSConfig{
+		CAFile:   opts.caFile,
+		CertFile: opts.certFile,
+		KeyFile:  opts.keyFile,
+	})
+	if err != nil {
+		return nil, err
+	}
+	transport := cleanhttp.DefaultPooledTransport()
```

Comment:
This disables keepalives, given we've had reports of consul running out of FDs that sounds like a bad idea.